spec/set_attributes_from_version_spec.rb

```diff
@@ -2,44 +2,65 @@
 
 describe 'java::set_attributes_from_version' do
   platforms = {
-    'centos-6.9' => {
+    'centos-6.9-jdk6' => {
```

Comment:
This isn't really the way we usually do things, platform is an attribute from ohai that we want to keep the way it is. 

Let's remove the `-jdk...`suffix from all these.
```suggestion
    'centos-6.9' => {
```